roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: restart openshift-master
-  service: name=openshift-master state=restarted
+- name: restart master
+  service: name={{ product_type }}-master state=restarted
```

Comment:
Does this actually work?  Here's what I'm seeing:

~~~
TASK: [openshift_master | Create the policy file if it does not already exist] ***
changed: [ose3-master.example.com]
ERROR: change handler (restart {{ product_type }}-master) is not defined
~~~

I have the following in my inventory:

~~~
deployment_type=enterprise
product_type=openshift
~~~